packages/core/src/types.ts

```diff
@@ -331,6 +331,7 @@ export enum Clients {
 export type Character = {
     id?: UUID; // optional UUID which can be passed down to identify the character
     name: string;
+    username: string;
```

Comment:
is this twitter specific?

Reply:
not only it can also be used as a nickname for the agent and its an index in memories